dotCMS/src/integration-test/java/com/dotcms/concurrent/scheduler/DotSchedulerImplTest.java

```diff
@@ -0,0 +1,91 @@
+package com.dotcms.concurrent.scheduler;
+
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.concurrent.DotConcurrentFactory;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UUIDGenerator;
+import com.github.kagkarlsson.scheduler.Scheduler;
+import com.github.kagkarlsson.scheduler.task.helper.OneTimeTask;
+import com.github.kagkarlsson.scheduler.task.helper.Tasks;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class DotSchedulerImplTest {
+
+    private final DotScheduler dotScheduler = DotConcurrentFactory.getInstance().getScheduler();
+
+    private final OneTimeTask<DotTask> oneTime = Tasks
+            .oneTime(
+                    "one-time-task",
+                    DotTask.class)
+            .execute(dotScheduler::executeOnce);
+
+    @CloseDBIfOpened
+    @BeforeClass
+    public static void prepare() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+        new DotConnect().setSQL("delete from scheduled_tasks").loadResult();
+    }
+
+    /**
+     * This method starts another scheduler (like one running on another server)- along with the
+     * scheduler embedded in the SchedulerAPI to ensure that Dot Synchronized tasks run in series, one at
+     * a time
+     *
+     * @throws InterruptedException
+     */
+    @Test
+    public void test_sync_task() throws Exception {
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)